pkg/operation/botanist/component/vpa/templates/crd-verticalpodautoscalers.tpl.yaml

```diff
@@ -1,7 +1,9 @@
 ---
-# For backwards compatibility it's not possible to take the community based VPA definition since it tightens
-# the validation too much. Instead, the CRD needs to retain and permit unknown fields as it used to be with
-# the `v1beta1` CRD version.
+# Source: https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml
```

Comment:
You could rather pin to the current VPA version we use `0.11.0` instead of master. We don't know how this file will look like in tomorrow's master.

```suggestion
# Source: https://github.com/kubernetes/autoscaler/blob/vertical-pod-autoscaler-0.11.0/vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml
```

The same is applicable for the `verticalpodautoscalercheckpoints` CRD - ref https://github.com/gardener/gardener/blob/90044fd673d5fae6b550ed70e1d5e49aaa551aba/pkg/operation/botanist/component/vpa/templates/crd-verticalpodautoscalercheckpoints.tpl.yaml#L1-L2.

Reply:
Done. Now pinned to "0.11.0".